### ISSUE TITLE
[FR] A clear and concise title describing the solution you'd like

### ISSUE THREAD

**Issue by sriprat5445** [enricocid/VectorifyDaHome#31]
1) **Describe the solution you'd like**
Notes:
- only ONE feature
- don't ask black theme or a different colors scheme as we have to follow guidelines for accessibility.

2) **Describe alternatives you've considered**

3) **Additional context**

**Comment by sriprat5445** [comment]
good
